### PR TITLE
Upgrade Camel Quarkus to 1.4.0

### PR DESCRIPTION
**Pull request by jamesnetherton**

### CODE REVIEW

File: integration-tests/camel/invoked/root/camel-debezium/pom.xml [hunk 1]
```diff
@@ -10,6 +10,11 @@
 
     <artifactId>quarkus-universe-integration-tests-camel-debezium</artifactId>
 
+    <!-- TODO: Remove when all platform participants (i.e debezium) can work with testcontainers >= 1.15.0 -->
```
Comment (gsmet) [inline integration-tests/camel/invoked/root/camel-debezium/pom.xml:13]: Have you reported this issue to the Debezium team? /cc @gunnarmorling @Naros

Comment (jamesnetherton) [reply integration-tests/camel/invoked/root/camel-debezium/pom.xml:13]: Seems that Debezium 1.4 aligns with testcontainers 1.15.x. So when that eventually makes it into platform, I can remove the workaround.

Comment (gsmet) [reply integration-tests/camel/invoked/root/camel-debezium/pom.xml:13]: OK, thanks!

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**